english/malaria/malaria.md

```diff
@@ -1,21 +1,23 @@
 The hand of marshmallow is speeding towards you and growing as it approaches
 . You start running away from it, but it seems that the walls are closing
  in on you. Suddenly everthing stops and dead silence ensues. You turn
-around to take a look at the marshmallow hand and in a flash you are 
-enveloped in a sea of marshmallow. Wherever you see, there is nothing but 
-marshmallow. You flail and trash but to no avail. You cannot draw breath. 
-You are suffocating! It starts getting dark to your eyes. 
+around to take a look at the marshmallow hand and in a flash you are
+enveloped in a sea of marshmallow. Wherever you see, there is nothing but
+marshmallow. You flail and trash but to no avail. You cannot draw breath.
+You are suffocating! It starts getting dark to your eyes.
 There is no water near.
 
 You start to [scream](scream/scream.md)
 
-You jump up from the reverie and find yourself by a forest road with a 
-bottle of pills by your side. You see a jeep crashed into the trees some 
+You jump up from the reverie and find yourself by a forest road with a
+bottle of pills by your side. You see a jeep crashed into the trees some
 distance away. The sound of distant gunfire reaches your ears and it dawns
 on you that you are near the town of Impala.
 
 Now stop fooling around and take your malaria pill. You've got to find the
 jackal and put an end to this madness. How else do you think you're
 going to complete FarCry 2. Huh?
 
+Malaria is a parasitic disease transmitted by the Anopheles mosquito.  You should use mosquito repellent and sleep under a bed net to reduce your risk of becoming infected.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.